Branch-SDK/src/androidTest/java/io/branch/referral/BranchSDKTests.java

```diff
@@ -234,32 +197,32 @@ public void onStateChanged(boolean changed, BranchError error) {
                 signal.countDown();
             }
         });
-        signal.await(1, TimeUnit.SECONDS);
+        Assert.assertTrue(signal.await(TEST_TIMEOUT, TimeUnit.MILLISECONDS));
     }
 
     @Test
     public void testZLoad() throws InterruptedException {
-        initSession();
-
+        initSessionResumeActivity();
         final CountDownLatch signalFinal = new CountDownLatch(1);
         final CountDownLatch signal = new CountDownLatch(1);
-        for (int i = 0; i < 100; i++) {
-            final int idx = i;
+        final int reps = 20;
```

Comment:
this used to be a much larger number, like 100, which just bogged up resources as each request
needs it's own AsyncTask, and the test never actually tested that all 100 requests succeeded because `signal.await(10, TimeUnit.SECONDS);` did not have `Assert.assertTrue` in front of it